deployments/bscmainnet_addresses.json

```diff
@@ -116,7 +116,7 @@
     "XVSStore": "0x1e25CF968f12850003Db17E0Dba32108509C4359",
     "XVSVault": "0xEA456C6a52c36Ae021D93cf69812260149Ec39c2",
```

Comment:
I think we should remove the file `venus-protocol/deployments/bscmainnet/XVSVault.json`, because that name (`XVSVault`) is not used anymore for the implementation of the vault. We are using `XVSVaultProxy_Implementation` instead

Reply:
Done https://github.com/VenusProtocol/venus-protocol/pull/476/commits/84221d79882da3df7617db59dd90bc0ef6705f6d